### ISSUE TITLE
Schema cannot be added to registry object 

### ISSUE THREAD

**Issue by daviskirk** [pyeve/cerberus#354]
Used Cerberus 1.1:

- I consulted these documentations:
  - [x] http://docs.python-cerberus.org/en/stable/
  - [x] http://docs.python-cerberus.org/en/latest/

- I consulted these sections of the docs (add more lines as necessary):
  - http://docs.python-cerberus.org/en/latest/schemas.html#registries

- [x] I found nothing relevant to my problem in the docs.
- [x] I found the documentation not helpful to my problem.

- [x] I have the capacity to improve the docs when my problem is solved.
- [x] I have the capacity to submit a patch when a bug is identified.

----

#### Use-case abstract

I am trying to add schemas to a custom registry so that I don't clutter the `cerberus.schema_registry` namespace. 
Adding any schema to a newly created `Registry` object results in an attribute error.

----

#### Support request / Bug report

```python
import cerberus 
r = cerberus.Registry()
r.add('non-system user', {'uid': {'min': 1000, 'max': 0xffff}})
```
results in 
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-20-ad8c4bb8a5d2> in <module>()
----> 1 r.add('non-system user', {'uid': {'min': 1000, 'max': 0xffff}})

~/anaconda/envs/athion-forecaster/lib/python3.6/site-packages/cerberus/schema.py in add(self, name, definition)
    375         :param definition: The definition.
    376         :type definition: any :term:`mapping` """
--> 377         self._storage[name] = self._expand_definition(definition)
    378 
    379     def all(self):

AttributeError: 'Registry' object has no attribute '_expand_definition'
```

**Comment by funkyfuture** [comment]
Have you checked the current master branch? there was a fix reg. the registry, though the error message you posted wasn't reported yet afaict.

**Comment by daviskirk** [comment]
No, I'll check the master branch and report back

**Comment by funkyfuture** [comment]
@daviskirk could you check this in the meantime?

**Comment by pib** [comment]
`Registry` is the abstract parent class of `SchemaRegistry` and `RulesSetRegistry` which are not exported from the top-level `cerberus` module.

I solved this same problem for myself by `from cerberus.schema import SchemaRegistry` and then using that.

**Comment by funkyfuture** [comment]
addressed in #383.
